darshan-runtime/doc/darshan-runtime.txt

```diff
@@ -122,8 +122,10 @@ make install
 ** NOTE: Users must call `--enable-hdf5-mod` to enable HDF5 modules, `--with-hdf5` is only used to additionally provide an HDF5 install prefix.
 ** NOTE: HDF5 instrumentation only works on HDF5 library versions >=1.8, and further requires that the HDF5 library used to build Darshan and the HDF5 library being linked in either both be version >=1.10 or both be version <1.10
 ** NOTE: This option does not work with the profile configuration instrumentation method described in the "Instrumenting applications" section (link:darshan-runtime.html#_using_a_profile_configuration[Using a profile configuration])
-* `--disable-pnetcdf-mod`: disables compilation and use of Darshan's PNetCDF
-  module (default=enabled)
+* `--enable-pnetcdf-mod`: enables compilation and use of Darshan's PnetCDF
+  module (default=disabled)
+** NOTE: This option requires the PnetCDF install prefix as an argument (e.g., `--with-pnetcdf=/path/to/pnetcdf/`)
```

Comment:
We can probably relax this requirement similar to how we do with the HDF5 module. In Cray build environments (and maybe also in Spack?), the PnetCDF installation can probably be auto-detected so that users don't have to explicitly provide the prefix, just by having the module loaded.

Reply:
I will have a look at this one to see if I can resolve this.

Reply:
Changing the default to "auto" makes sense and is OK with me.